samples/tutorial/Tutorial4.md

```diff
@@ -224,48 +260,23 @@ object TodoWorkflow : StatefulWorkflow<TodoProps, State, Back, List<Any>>() {
     // When a todo item is selected, edit it.
     state = state.copy(step = Step.Edit(index))
   }
-
-  private fun newTodo() = action {
-    // Append a new todo model to the end of the list.
-    state = state.copy(
-        todos = state.todos + TodoModel(
-            title = "New Todo",
-            note = ""
-        )
-    )
-  }
 }
 ```
 
-Update the `RootWorkflow` to defer to the `TodoWorkflow` for rendering the `Todo` state. This will get us back into a state where we can build again (albeit without editing support):
+So far `RootWorkflow` is still deferring to the `TodoListWorkflow`. Update the `RootWorkflow` to defer to the `TodoWorkflow` for rendering the `Todo` state. This will get us back into a state where we can build again (albeit without editing support):
 
 ```kotlin
-object RootWorkflow : StatefulWorkflow<Unit, State, Nothing, BackStackScreen<*>>() {
+object RootWorkflow : StatefulWorkflow<Unit, State, Nothing, BackStackScreen<Any>>() {
 
   // …
 
   override fun render(
     renderProps: Unit,
     renderState: State,
     context: RenderContext
-  ): BackStackScreen<*> {
-
-    // Our list of back stack items. Will always include the "WelcomeScreen".
-    val backstackScreens = mutableListOf<Any>()
+  ): BackStackScreen<Any> {
 
-    // Render a child workflow of type WelcomeWorkflow. When renderChild is called, the
-    // infrastructure will create a child workflow with state if one is not already running.
-    val welcomeScreen = context.renderChild(WelcomeWorkflow) { output ->
-      // When WelcomeWorkflow emits LoggedIn, turn it into our login action.
-      login(output.username)
-    }
-    backstackScreens += welcomeScreen
-
-    when (state) {
-      // When the state is Welcome, defer to the WelcomeWorkflow.
-      is Welcome -> {
-        // We always add the welcome screen to the backstack, so this is a no op.
-      }
+      // …
```

Comment:
I replaced this unchanged code with `//...` to draw more attention to the code that is updated.